ui/src/App.svelte

```diff
@@ -74,7 +73,7 @@
     activeWarrior = w;
   });
 
-  const detectedLocale = activeWarrior.locale || detectLocale();
```

Comment:
removing detectLocale removes the ability for the users browser to tell the app what locale to use, so for instances where a default locale is not really intended to be enforced this degrades the experience.  I would suggest perhaps doing a check for defaultLocale != en then use it otherwise use detectLocale.

Reply:
I went back a checked by I originally removed detectLocale() and turns out in the current implementation of detectLocale(), there is no locale detectors configured so it always returned "en".
I have now fixed this by adding the navigatorDetector (https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/detectors#navigator) and use this in preference over the DefaultLocale.

Order of importance is now:
Userprofile-locale > Browser's preffered locale > DefaultLocale